libs/books/data-access/src/lib/+state/reading-list.reducer.ts

```diff
@@ -47,11 +47,22 @@ const readingListReducer = createReducer(
       error: action.error
     };
   }),
-  on(ReadingListActions.addToReadingList, (state, action) =>
-    readingListAdapter.addOne({ bookId: action.book.id, ...action.book }, state)
+  on(ReadingListActions.addToReadingList, (state, action) =>{
+    //console.log(action, state);
+    return readingListAdapter.addOne({ bookId: action.book.id, ...action.book }, state)}
   ),
-  on(ReadingListActions.removeFromReadingList, (state, action) =>
-    readingListAdapter.removeOne(action.item.bookId, state)
+  on(ReadingListActions.removeFromReadingList, (state, action) =>{
+    
+    return readingListAdapter.removeOne(action.item.bookId, state)}
+  ),
+////////////////////////////////////////////////////////////////////
+  on(ReadingListActions.failedAddToReadingList, (state, action) =>{
+    //console.log(action, state);
+    return readingListAdapter.removeOne(action.book.id, state)}
```

Comment:
Pick a style and stick with it, please. In this case, try to match the prevailing style. i.e. `=> readingListAdapter...` as opposed to `=> { return readingListAdapter...`